src/overlays/actors/ovl_En_Niw/z_en_niw.h

```diff
@@ -5,11 +5,97 @@
 
 struct EnNiw;
 
+typedef void (*EnNiwActionFunc)(struct EnNiw*, GlobalContext*);
+
+typedef struct {
+    // in OOT this is "type" and there is one instance of an if
+    //  statement specifying `if (true) if (enabled == 1)`
+    //  so in beta there might have been a third value
+    /* 0x0000 */ u8 enabled;
+    /* 0x0004 */ Vec3f pos;
+    /* 0x0010 */ Vec3f vel;
+    /* 0x001C */ Vec3f accel;
+    /* 0x0028 */ s16 life;
+    /* 0x002A */ s16 zRotStart;
+    /* 0x002C */ f32 scale;
+    /* 0x0030 */ f32 zRot;
+    /* 0x0034 */ u8 timer;
+} EnNiwFeather; // size = 0x38
+
+#define ENNIW_LIMBCOUNT 0x10
+
 typedef struct EnNiw {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x678];
+    /* 0x144 */ SkelAnime skelanime;
+    /* 0x188 */ Vec3s limbDrawTbl[ENNIW_LIMBCOUNT];
+    /* 0x1E8 */ Vec3s transitionDrawtable[ENNIW_LIMBCOUNT];
+    /* 0x248 */ EnNiwActionFunc actionFunc;
+    /* 0x24C */ s16 unkTimer24C;
+    /* 0x24E */ s16 unkTimer24E;
+    /* 0x250 */ s16 unkTimer250;
+    // 252 is re-used, after thrown is used as a hop timer
+    /* 0x252 */ s16 unkTimer252;
```

Comment:
```suggestion
    // 252 is re-used, after thrown is used as a hop timer
    /* 0x252 */ s16 unkTimer252;
```
If this timer is used for two different purposes, then you can use a `union` to represent that. But we can also leave that for later documentation, it's not a priority

Reply:
```suggestion
    /* 0x252 */ s16 unkTimer252;  // 252 is re-used, after thrown is used as a hop timer
```